### ISSUE TITLE
deps: cannot contain more than 1 space to separate dependencies

### ISSUE THREAD

**Issue by felipealmeida** [vtorri/ewpi#23]
The current parser doesn't ignore all spaces between dependencies, which causes the ewpi to consider an empty string a dependency when there are multiple spaces between dependencies in the ewpi file.

deps:^dep1^^dep2

Causes the ewpi to understand that there are three dependencies, namely: "dep1", "", "dep2". Which causes crashes.

**Comment by felipealmeida** [comment]
Not a very important bug, but leaves users guessing when they tinker around. But since the ewpi files are defined by the project itself, I'd say this is very low priority.

**Comment by vtorri** [comment]
indeed, i have coded quickly the parser and i have decided that there should ne one space only. I'm aware of that. as well as no space between 'deps' and ':'. Also, no space after "deps:" if no  dependencies.

**Comment by felipealmeida** [comment]
I've tried to fix it real fast but it failed. I'll see if I fix it later tonight. It took me a while to understand what was going on, so that's why it did bother a little bit, and as more people in Expertise will likely use it, I rather they do not fall on the same pitfalls I did.

**Comment by vtorri** [comment]
the parser shold just ignore spaces, i guess. This should let the rest of the code unchanged
